### PR TITLE
issue #1435 -- use 160 column line length lint and correct violations

### PR DESCRIPTION
**Pull request by suprjinx**

### CODE REVIEW

**Review by kannon92 — COMMENTED** [review]
Hey Geoffrey!  Thanks for your first PR!  This looks good to me.  I will make sure tests pass before giving final approval!

**Review by kannon92 — CHANGES_REQUESTED** [review]

File: internal/pulsarutils/async.go [hunk 1]
```diff
@@ -34,11 +34,21 @@ var log = logrus.NewEntry(logrus.StandardLogger())
 
 // Receive returns a channel containing messages received from pulsar.  This channel will remain open until the
 // supplied context is closed.
-// consumerId: Internal Id of the consumer.  We use this so that when messages from different consumers are multiplexed, we know which messages originated form which consumers
+// consumerId: Internal Id of the consumer.  We use this so that when messages from different consumers are
```
[the remark this answers is not in the document]
Comment (suprjinx) [reply internal/pulsarutils/async.go:37]: hmm yeah where'd that second line go... fixing

File: .golangci.yml [hunk 2]
```diff
@@ -1,3 +1,7 @@
+run:
+  skip-dirs:
+    - pkg/api
```
[the remark this answers is not in the document]
Comment (suprjinx) [reply .golangci.yml:3]: okay i'll narrow this -- I wonder if it's worth isolating the generated code better?

File: internal/pulsarutils/async.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (suprjinx) [reply internal/pulsarutils/async.go:37]: this file still looks correct on the branch, so something is funky with the diff:

https://github.com/suprjinx/armada/blob/line-length-lint/internal/pulsarutils/async.go

File: .golangci.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (suprjinx) [reply .golangci.yml]: just removed -- golangci passes locally without it so we'll see how CI does

[the remark this answers is not in the document]
Comment (kannon92) [reply .golangci.yml]: Yea I think that is a good idea for a follow up.

[the remark this answers is not in the document]
Comment (suprjinx) [reply .golangci.yml]: The docs say that golangci-lint detects and skips autogen files, so I think we can just leave the config as is

**Review by kannon92 — APPROVED** [review]

**Review by richscott — APPROVED** [review]
Much easier on the eyes!

**Review by jayofdoom — APPROVED** [review]

**Pull request merged**